net/node/node.go

```diff
@@ -12,6 +12,8 @@ import (
 	nnetpb "github.com/nknorg/nnet/protobuf"
 )
 
+var NodeRelayMessage uint64 = 0
```

Comment:
It's better make it a (private) field of LocalNode structure because conceptually it makes more sense (and works if we have multiple local nodes for example)